storage/database/db_manager.go

```diff
@@ -823,6 +824,22 @@ func (dbm *databaseManager) DeleteCanonicalHash(number uint64) {
 	dbm.cm.writeCanonicalHashCache(number, common.Hash{})
 }
 
+// ReadAllHashes retrieves all the hashes assigned to blocks at a certain heights,
+// both canonical and reorged forks included.
+func (dbm *databaseManager) ReadAllHashes(number uint64) []common.Hash {
```

Comment:
Where is this method used?

Reply:
@jeongkyun-oh 
Oh, this method will be used in `headerchain.go` **after** snapshot package is ported.